.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "function-benchmarks"]
+	path = function-benchmarks
+	url = https://github.com/ddps-lab/serverless-faas-workbench.git
+[submodule "openFaas/openFaas/pigo-openfaas"]
```

Comment:
I notcies two pigo-openfaas submodules, do you only need one of them? if so, which one?

Reply:
dont need the function benchmarks submodule. I thought I removed it, but ti shouldnt harm anything. The pigo one not sure why there are two cause I only created it once. I dont think it should break anything tho

Reply:
Ok I'll merge and delete if needed